### PR TITLE
CHORE: Final tweaks for v1.0.0 release

### PR DESCRIPTION
**Pull request by TomWinder**
## What is the purpose of this Pull Request? 
Final tweaks before the v1.0.0 release. This includes:
 * change the method parameter name from `1dsweep` to `1dnlloc` for NonLinLoc Grid2Time LUT calculation (b3c0c48)
 * consequent updates to the LUT tutorial, plus some layout improvements, update of LCC standard parallels example/ description (b55d612)
 * docs/index.rst: update to match the main GitHub README (8203f3b)
 * installation instructions: update NonLinLoc installation guide; improve instructions for macOS; add instructions to run the dike example / test (8203f3b)
 * chores: updating copyright notices, package descriptions etc. so they are consistent throughout (7d5d820)
 * changed default thread count to 1 in the jupyter notebook examples; this is mainly for smooth integration with binder. Also updated the Rutford settings to (a) match the paper and (b) make it fit within the binder memory requirements (bd66c14)
 * Changed default thread count in all examples to 4 (more appropriate for most users) with clear note to increase if possible (b0188a3)
 * changed default onset settings in template scripts to more widely appropriate values, plus tweaks to local_mag, trigger settings / comments. (83302dc)

## Test cases
Tested on all examples, and by building and running on binder.

- [x] All examples run without any new warnings
- [x] test_benchmarks.py reports all example tests pass

## TO-DO:
- [x] Add binder badge / links. Currently unsure how to do this before pushing to master.
- [x] Add changelog.

### System details
Ubuntu 18.04 / python 3.8
Binder

### Final checklist
- [x] `master` base branch selected?
- [x] All tests still pass.
- [x] Any new features or fixed regressions are covered by new tests.
- [x] Any new or changed features are fully documented.
- [x] Significant changes have been added to `CHANGES.md`.

### CODE REVIEW

File: quakemigrate/lut/create_lut.py [hunk 1]
```diff
@@ -230,7 +259,7 @@ def _compute_1d_fmm(lut, phase, vmodel):
 
     Parameters
     ----------
-    lut : :class:`~quakemigrate.lut.LUT` object
+    lut : :class:`~quakemigrate.lut.lut.LUT` object
```
Comment (hemmelig) [inline quakemigrate/lut/create_lut.py:262]: Yeah, this is something I was unsure of when initially adding many of these - I think either is fine, so long as we are consistent. I used the former representation as, for example, `numpy.ndarray` is used very commonly, despite the actual class definition being under `numpy.core`. On the other hand, much of the obspy documentation provides full paths.

File: docs/conf.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
 # -- Project information -----------------------------------------------------
 project = 'QuakeMigrate'
-copyright = '2020, QuakeMigrate developers'
+copyright = '2020-2021, QuakeMigrate developers'
```
Comment (hemmelig) [inline docs/conf.py:25]:
```suggestion
 copyright = '2020–2021, QuakeMigrate developers'
```
en dash - really a question of our pedanticism...

File: quakemigrate/lut/create_lut.py (hunk 1 above)
Comment (TomWinder) [reply quakemigrate/lut/create_lut.py:262]: For local functions, readthedocs doesn't successfully make a link unless you use the full version (have experimented). (e.g. look at the Return type: LUT object [here](https://quakemigrate.readthedocs.io/en/latest/submodules/quakemigrate.lut.html#quakemigrate.lut.create_lut.compute_traveltimes))

Comment (hemmelig) [reply quakemigrate/lut/create_lut.py:262]: Great, then that settles it. Worth doing a package-wide sweep?

Comment (TomWinder) [reply quakemigrate/lut/create_lut.py:262]: Probably. I've been trying to do it in each file I've edited over the past couple of months, but there will defo be some still loitering about.

File: quakemigrate/lut/create_lut.py [hunk 3]
```diff
@@ -151,41 +169,52 @@ def compute_traveltimes(grid_spec, stations, method, phases=["P", "S"],
...
+                      DeprecationWarning, 2)
+        method = "1dnlloc"
+
     if method == "homogeneous":
         logging.info("Computing homogeneous traveltimes for...")
         lut.velocity_model = "Homogeneous velocity model:"
         for phase in phases:
             velocity = kwargs.get(f"v{phase.lower()}")
             if velocity is None:
-                raise TypeError(f"Missing argument: 'v{phase.lower()}'")
```
[the remark this answers is not in the document]
Comment (TomWinder) [reply quakemigrate/lut/create_lut.py:160]: Ok, hadn't looked at "accepted practice" - this was a slightly bleary-eyed change, tbh. Seems `TypeError` if it's not specified (so None) or `ValueError` if it's not an acceptable option - though this feels slightly strange to use for an incorrect `compute_traveltimes()` method..

File: docs/conf.py (hunk 2 above)
Comment (QuakeMigrate-developers) [reply docs/conf.py:25]: We'll do this in 2022 - not worth the faff.

Comment (TomWinder) [reply docs/conf.py:25]: We'll do this in 2022. Not worth the faff

File: quakemigrate/lut/create_lut.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (TomWinder) [reply quakemigrate/lut/create_lut.py]: Done

File: quakemigrate/lut/create_lut.py (hunk 1 above)
Comment (TomWinder) [reply quakemigrate/lut/create_lut.py:262]: (Done)

**Review by hemmelig — APPROVED** [review]
Approving.

**Pull request merged**